### DISCUSSION TITLE
Custom sources that include structured sections should be possible to reorder and have their own order

### DISCUSSION THREAD

**Discussion by outline4** [craftcms/cms#11184] [Ideas]
**Custom sources that include structured sections should be possible to reorder and have their own order vs the global order.** 

I am referring to [this thread](https://github.com/craftcms/cms/issues/11163)

There should be a global order of structured elements and an order per custom source. Maybe this concept could be enlarged to filters also: each filter has its custom order. 

### To clarify:

Some of our customers needs to sort entries based on categories. This also applies to a lot of craft commerce customers we have.  I was so happy to hear the rise of custom structures, where I thought this was possible.

### For instance: 

in certain shops you list all entries (without any category) in a certain order, and when applying a category, in a different order. Such a thing was possible with Low Reorder in EE and was highly appreciated by our clients.

**Comment by brandonkelly** [comment]
This idea has actually come up internally before. We could do away with Structure sections entirely, and replace them with structured views.

**Reply by thupsi** [reply]
@brandonkelly This sounds great! Is it related to what you where hinting at in https://github.com/craftcms/cms/discussions/9783#discussioncomment-1274453? Combined with some other minor things it could be a great solution for including whole sections AND the section index in a single source, and would open many other possibilities for enhancing the way content is presented in the CP. 

I also remember that in some other issue you mentioned that you where thinking of a "content view", I'm not sure if that is the same thing, I can't find it right now.

**Reply by brandonkelly** [reply]
Different but I suppose related :)

**Comment by outline4** [comment]
That sounds interesting.  
Structured custom sources is definitely something we are looking for.  
Is that on a timeframe somehow, somewhere? 

Separate order for custom sources: Is that something you consider doing, or not? 

Cheers  
Stefan

**Reply by brandonkelly** [reply]
It’s the sort of thing we’d do for a major release, not a point release. Can’t say more than that.
